third_party/terraform/website/docs/r/google_service_account.html.markdown

```diff
@@ -10,6 +10,11 @@ description: |-
 
 Allows management of a [Google Cloud Platform service account](https://cloud.google.com/compute/docs/access/service-accounts)
 
+-> Creation of service accounts is eventually consistent, and that can lead to
+errors when you try to apply IAM roles to service accounts immediately after
```

Comment:
I've never met this issue with IAM policies and am using them quite extensively.

Could this be an issue only with ACLs?

Reply:
Thanks! When writing that out I accidentally switched from ACL -> IAM. Updated.